scripts/update_scpca_checkpoints.py

```diff
@@ -0,0 +1,479 @@
+        action="store_true",
+        help = "overwrite existing files at destination"
+    )
+    const = parser.add_argument_group('Common Values')
+    const.add_argument(
+        "--ref_assembly",
+        default = "Homo_sapiens.GRCh38.104",
+        help = "reference assembly"
+    )
+    const.add_argument(
+        "--ref_fasta",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/fasta/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz",
+        help = "path or uri to genome reference fasta"
+    )
+    const.add_argument(
+        "--ref_gtf",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz",
+        help = "path or uri to genome reference gtf"
+    )
+    const.add_argument(
+        "--salmon_splici_index",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_intron.txome",
+        help = "path or uri to splici index"
+    )
+    const.add_argument(
+        "--salmon_bulk_index",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_cdna.txome",
+        help = "path or uri to bulk index"
+    )
+    const.add_argument(
+        "--t2g_3col_path",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.spliced_intron.tx2gene_3col.tsv",
+        help = "path or uri to 3 column t2g file for splici index"
+    )
+    const.add_argument(
+        "--t2g_bulk_path",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.spliced_cdna.tx2gene.tsv",
+        help = "path or uri to bulk t3g file"
+    )
+    const.add_argument(
+        "--cellranger_index",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/cellranger_index/Homo_sapiens.GRCh38.104_cellranger_full",
+        help = "path or uri to Cell Ranger index"
+    )
+    const.add_argument(
+        "--star_index",
+        default = "s3://scpca-references/homo_sapiens/ensembl-104/star_index/Homo_sapiens.GRCh38.104.star_idx",
+        help = "path or uri to STAR index"
+    )
+    const.add_argument(
+        "--nextflow_version",
+        default = "22.10.0",
+        help = "default nextflow version"
+    )
+    const.add_argument(
+        "--scpca_version",
+        default = "v0.2.0",
+        help = "default scpca version"
+    )
+    const.add_argument(
+        "--barcode_dir",
+        default = "s3://scpca-references/barcodes/10X",
+        help = "path or uri to directory with barcode files"
+    )
+    args = parser.parse_args()
+
+    ### Read in library file
+    library_df = pandas.read_csv(args.library_file, sep = "\t",  dtype = 'string', keep_default_na=False)
+
+    # add barcode files
+    library_df['barcode_file'] = library_df['technology'].apply(get_barcode, barcode_dir = args.barcode_dir)
+
+    library_df = library_df.iloc[371:373] # limit to 2 for testing
```

Comment:
Just noting that this should be removed before merging/ running it through for real.